src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -41,7 +70,7 @@ public sealed class OptionalFieldAttribute : Attribute
     {
         public int VersionAdded
         {
-            get { return default(int); }
+            get { throw null; }
```

Comment:
I've never seen us throw in the contracts.  Is this something we're going to be doing everywhere now?